static/css/custom_reporting_dark.css

```diff
@@ -0,0 +1,209 @@
+.some-page-wrapper {
+  margin: 15px;
+  background-color: #3f454a;;
+}
+.row {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+}
+.column {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100%;
+  flex: 1;
+}
+.row-divider {
+  background-color: #efefef;
+  border-width: 1px;
+  border-style: solid;
+  border-color: #add3e6;
+  border-radius: 10px;
+  height: 30px;
+  margin: 5px;
+  text-align: center;
+  font-weight: bold;
+}
+
+/* DASHBOARD */
+
+.white {
+  background-color: white;
+}
+
+.green {
+  background-color: #d9fce9;
+}
+
+.orange {
+  background-color: #f8e9d0;
+}
+
+.red {
+  background-color: #fff9fa;
+}
+
+.blue {
+  background-color: #ebfbfb;
+}
+
+.row-1line-title:hover + .hide{
+  display: block;
+}
+
+.row-1line-title {
+  border-width: 1px;
+  border-style: solid;
+  border-color: #add3e6;
+  border-radius: 10px;
+  height: 100px;
+  margin: 5px;
+  text-align: center;
+  padding: 25px;
+  padding-top: 35px;
+  padding-bottom: 35px;
+  font-weight: bold;
+  font-size: 100%;
+}
+
+.row-2line-title:hover + .hide{
+  display: block;
+}
+
+.row-2line-title {
+  border-width: 1px;
+  border-style: solid;
+  border-color: #add3e6;
+  border-radius: 10px;
+  height: 100px;
+  margin: 5px;
+  text-align: center;
+  padding: 25px;
+  font-weight: bold;
+  font-size: 100%;
+}
+
+.hide{
+  display: none;
+  position: absolute;
+  background-color: #ebfbfb;
+  border-width: 1px;
+  border-style: dashed;
+  border-color: #add3e6;
+  border-radius: 10px; 
+  height: 50px;
+  margin: 5px;
+  text-align: center;
+  text-decoration: none;
+  padding: 10px;
+  font-size: 100%;
+  left: 25%;
+}
+
+.report-row {
+  border-width: 1px;
+  border-style: solid;
+  border-color: #add3e6;
+  border-radius: 10px;
+  height: 100px;
+  margin: 5px;
+  text-align: center;
+  padding: 30px;
```

Comment:
```suggestion

```

Reply:
i cant find a way to vertically align the what is in the column for flex css.
padding seems to be the only way to somehow make it look like its in the middle.

Reply:
changed the way its applied instead of delete